ee/clickhouse/sql/person.py

```diff
@@ -20,9 +20,9 @@
     created_at DateTime64,
     team_id Int64,
     properties VARCHAR,
-    is_identified Boolean,
-    is_deleted Boolean DEFAULT 0
+    is_identified Boolean
     {extra_fields}
+    , is_deleted Boolean DEFAULT 0
```

Comment:
Something less than ideal. We forgot to specify adding the `is_deleted` column before the kafka columns so now on production `is_deleted` comes after the kafka columns but our schema here specified creating the columns in a different order so insertions were acting up because ordering was different